stable/seq/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: seq
+version: 0.1.0
+appVersion: 5.0.832-pre
+description: Seq is the easiest way for development teams to capture, search and visualize structured log events! This page will walk you through the very quick setup process.
+keywords:
+- seq
+- structured
+- logging
+home: http://getset.net/
+icon: https://avatars1.githubusercontent.com/u/5898109?s=200&v=4
+sources:
+- https://github.com/datalust/seq-tickets
+maintainers:
```

Comment:
Any reason why you don't put the other owners here?

Reply:
No real reason, I would be the one supporting this they maintain the seq images.